.editorconfig

```diff
@@ -15,6 +15,7 @@ indent_size = 4
 indent_style = space
 
 # New line preferences
+end_of_line = lf
```

Comment:
You need to specify crlf for `.sln`

Reply:
Doesn't this apply to all files, as it's in a `[*]` region?

Reply:
Oh right. Actually I'd like to see if we can use LF in the solution file as well 🤔